motion_blender-support.gemspec

```diff
@@ -0,0 +1,21 @@
+# -*- encoding: utf-8 -*-
+require File.expand_path('../motion/version', __FILE__)
+
+Gem::Specification.new do |spec|
+  spec.name        = "motion_blender-support"
+  spec.version     = MotionSupport::VERSION
+  spec.authors     = ['kayhide', "Thomas Kadauke"]
+  spec.email       = ['kayhide@gmail.com', "thomaspec.kadauke@googlemail.com"]
```

Comment:
Something went wrong here with my email address :)

It's thomas.kadauke@googlemail.com